roles/ruby_s/tasks/main.yml

```diff
@@ -25,3 +25,17 @@
   ansible.builtin.include_tasks: install_from_source.yml
   when:
     - (installed_ruby.stderr is search("not found")) or (installed_ruby.stdout | regex_search('\d+\.\d+\.\d+') != ruby_version | regex_search('\d+\.\d+\.\d+'))
+
+- name: ruby_s | install global bundler, specific version
+  community.general.gem:
+    name: bundler
+    state: present
+    version: "{{ bundler_version }}"
+    user_install: false
+  when: bundler_version is defined
```

Comment:
community gem will use the Operating Systems ruby which we go not want.

Reply:
Switched back to ansible.builtin.command